flake.nix

```diff
@@ -1,55 +1,72 @@
 {
   description = "flora";
   inputs = {
-    flake-utils = { url = "github:numtide/flake-utils"; };
-    horizon-platform = {
-      url = "git+https://gitlab.horizon-haskell.net/package-sets/horizon-platform";
-    };
+    flake-utils.url = "github:numtide/flake-utils";
+    horizon-platform.url =
+      "git+https://gitlab.horizon-haskell.net/package-sets/horizon-platform";
     nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
-    streamly.url = "git+https://github.com/composewell/streamly";
-    streamly.flake = false;
-    poolboy.url = "git+https://github.com/blackheaven/poolboy";
+    nixpkgs-souffle_2_2.url = "github:nixos/nixpkgs/nixos-22.05";
```

Comment:
I'd name this `nixpkgs-2211` or `nixpkgs-stable`

Reply:
Good idea! I've done that and documented the `input`.